adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -640,6 +677,14 @@ public void setAdyenPaymentServiceFactory(AdyenPaymentServiceFactory adyenPaymen
         this.adyenPaymentServiceFactory = adyenPaymentServiceFactory;
     }
 
+    public CommerceCartService getCommerceCartService() {
+        return commerceCartService;
+    }
+
+    public void setCommerceCartService(CommerceCartService commerceCartService) {
```

Comment:
dependencies should have @Required so that they fail on startup time rather than at runtime! Possibly this is an issue in general In this AddOn?